<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomSelect</class>
 <widget class="QWidget" name="RoomSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>681</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <item row="0" column="1">
     <widget class="QListView" name="listView_room">
      <property name="maximumSize">
       <size>
        <width>411</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QListView" name="listView_class">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="resizeMode">
       <enum>QListView::Fixed</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>180</y>
     <width>191</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_create_room">
      <property name="text">
       <string>创建房间</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_join_room">
      <property name="text">
       <string>加入房间</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="MyToolButtonMin" name="toolButton_min_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>0</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="MyToolButtonClose" name="toolButton_close_2">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>0</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>40</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_user_pic">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Rsc.qrc">:/rsc/user.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_name">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyToolButtonClose</class>
   <extends>QToolButton</extends>
   <header location="global">mycontrolwidgets.h</header>
  </customwidget>
  <customwidget>
   <class>MyToolButtonMin</class>
   <extends>QToolButton</extends>
   <header location="global">mycontrolwidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Rsc.qrc"/>
 </resources>
 <connections/>
</ui>
